pages/u/[username]/card.tsx

```diff
@@ -35,67 +28,26 @@ const ADDITIONAL_PROFILES_TO_LOAD = [
   "CBID2",
 ];
 
-interface CardProps {
-  username: string;
-  cards: DevCardProps[];
-}
-
-interface Params extends ParsedUrlQuery {
-  username: string;
-}
+export type UserDevStats = DbUser & DbListContributorStat;
 
 async function fetchUserData(username: string) {
-  if (!isValidUrlSlug(username)) {
-    throw new Error("Invalid input");
+  try {
+    if (!isValidUrlSlug(username)) {
```

Comment:
You could used the suggested `GitHubUserNameSchema` up top here as well. `isValidUrlSlug` accepts underscores, but GitHub usernames do not allow for them.